File: 백준/Bronze/10952. A＋B － 5/A＋B － 5.js
var fs = require('fs');
var input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
for (var i = 0; i < input.length; i++) {
  var a = parseInt(input[i].split(' ')[0]);
  var b = parseInt(input[i].split(' ')[1]);
  if (a == 0 && b == 0) {
    break;
  } else {
    console.log(a + b);
  }
}

File: 백준/Bronze/10952. A＋B － 5/A＋B － 5.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        StringTokenizer st;
        String s;
        while(!(s = br.readLine()).equals("0 0")) {
            st = new StringTokenizer(s);
            sb.append(Integer.parseInt(st.nextToken()) + Integer.parseInt(st.nextToken()) + "\n");
        }
        System.out.println(sb.toString().trim());
    }
}
